flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -188,6 +167,90 @@ def test_battery_solver_day_2(
         )
 
 
+@pytest.mark.parametrize("use_inflexible_device", [False, True])
+def test_battery_solver_day_3(
+    # add_battery_assets, add_inflexible_device_forecasts, use_inflexible_device
+    add_battery_assets,
+    add_inflexible_device_forecasts,
+    use_inflexible_device,
+):
+    """Check battery scheduling results for day 3, which is set up with
+    8 hours with negative prices, followed by 16 expensive hours.
+
+    The battery is expected not to exploit the mechanism of charging and discharging within a time period
+    taking advantage of the inverter losses (heat) to consume energy. Nevertheless, as the consumption and production
+    prices are equal, the battery follows an oscillating dynamic in periods with negative prices. Again, due to conversion
+    efficiencies, the battery will charge **less** and will be able to discharge **more** than what is actually stored.
```

Comment:
I didn't understand this formulation.

Reply:
I've updated the formulation, let me know if is better :smile: